functions/go/templater/README.md

```diff
@@ -0,0 +1,46 @@
+# Templater function
+
+This is an example of implementing a templater function.
+
+This example is written in `go` and uses the `kyaml` libraries for parsing the
```

Comment:
This is an implementation detail that should not be exposed to the user. Instead explain the use case, why would anyone want to use this function? Why is it useful?